RubiksCubeSolver.h

```diff
@@ -119,10 +120,13 @@ struct RubiksCubeSolver {
     void tryDownAntiClockwiseTurn(Trie &cubesSeen, std::queue<std::pair<RubiksCube,int> > &activeCubes,
                                   RubiksCube &currentCube, int moves, int currentCubeIndex);
     
+    std::string fwdPath, bkwdPath;
+    std::string retraceHalfPath (Trie cubesSeen, std::string currentCubeString);
```

Comment:
Both arguments should be references, especially Trie because its very expensive to create a new instance of it